dwave/gate/simulator/operation_generation.py

```diff
@@ -517,6 +742,24 @@ def generate_op(
         cython_imports += op.cython_header + "\n\n"
         cython_functions += op.cython_function + "\n"
 
+    op_set_definitions = [
+        ("measurement_computational_basis", 1, 0, dict(
+            operators=[measurement_0, measurement_1]
+        )),
+        ("dephase_noise", 1, 0, dict(
+            operators=[dephase0, dephase1]
+        )),
+        ("amplitude_dampening_noise", 1, 0, dict(
+            operators=[amp_damp0, amp_damp1]
+        )),
```

Comment:
Is it correct to say that you've added these to the operations, but they are currently not used in any way by the simulator, i.e., there's no accessible support for them yet?

Reply:
Yes that's right